roles/grafana_agent/tasks/main.yml

```diff
@@ -32,6 +32,23 @@
   when: ansible_pkg_mgr == "apt"
   register: keyrings_exists
 
+- name: Remove conflicting Grafana key file if exist
+  become: true
+  ansible.builtin.file:
+    path: /etc/apt/keyrings/grafana.key
+    state: absent
+  when: ansible_pkg_mgr == "apt"
+
+- name: Remove existing Grafana APT repository
```

Comment:
Will this remove and recreate the repo every time?  Is there a way to make this conditional?

Reply:
@dmick I agree this is a bit bruteforce-y but it does prevent us from having to maintain some sort of sha or verification _about_ the key.  What is your concern?  Security?  Messing with apt cache?